.eslintrc.yaml

```diff
@@ -1,16 +1,29 @@
-extends: 'airbnb'
+env:
+  node: true
+
+extends:
+  - eslint:recommended
+  - airbnb
+
+globals:
+  uneval: false
+
 plugins:
   - mozilla
 
+root: true
+
 rules:
-  indent: [1, 2, {SwitchCase: 1}]
-  mozilla/balanced-listeners: [0]
-  mozilla/no-aArgs: [1]
-  mozilla/this-top-level-scope: [1]
-  no-console: [1]
-  import/no-unresolved: [0]
+  import/newline-after-import: off
+  import/no-extraneous-dependencies: off
+  import/no-unresolved: off
```

Comment:
One was already disabled. I can fix the 3 warnings caused by `import/newline-after-import`.
But fixing [`import/no-extraneous-dependencies`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md) seems like it's not worth the effort because it's choking on the `require('sdk')` style requires which aren't in package.json:

```
✗ npm run lint

> security-advisor@0.0.1 lint /Users/pdehaan/dev/github/mozilla/security-advisor-shield-study
> eslint lib data *.js


/Users/pdehaan/dev/github/mozilla/security-advisor-shield-study/lib/Advisor.js
   1:16  warning  'chrome' should be listed in the project's dependencies. Run 'npm i -S chrome' to add it  import/no-extraneous-dependencies
   5:26  warning  'sdk' should be listed in the project's dependencies. Run 'npm i -S sdk' to add it        import/no-extraneous-dependencies
   6:17  warning  'sdk' should be listed in the project's dependencies. Run 'npm i -S sdk' to add it        import/no-extraneous-dependencies
   7:17  warning  'sdk' should be listed in the project's dependencies. Run 'npm i -S sdk' to add it        import/no-extraneous-dependencies
   8:21  warning  'sdk' should be listed in the project's dependencies. Run 'npm i -S sdk' to add it        import/no-extraneous-dependencies
   9:23  warning  'sdk' should be listed in the project's dependencies. Run 'npm i -S sdk' to add it        import/no-extraneous-dependencies
  10:14  warning  'sdk' should be listed in the project's dependencies. Run 'npm i -S sdk' to add it        import/no-extraneous-dependencies
  11:21  warning  'sdk' should be listed in the project's dependencies. Run 'npm i -S sdk' to add it        import/no-extraneous-dependencies
  13:20  warning  'lib' should be listed in the project's dependencies. Run 'npm i -S lib' to add it        import/no-extraneous-dependencies
  14:26  warning  'lib' should be listed in the project's dependencies. Run 'npm i -S lib' to add it        import/no-extraneous-dependencies

/Users/pdehaan/dev/github/mozilla/security-advisor-shield-study/lib/Button.js
  1:24  warning  'chrome' should be listed in the project's dependencies. Run 'npm i -S chrome' to add it  import/no-extraneous-dependencies
  2:29  warning  'sdk' should be listed in the project's dependencies. Run 'npm i -S sdk' to add it        import/no-extraneous-dependencies
  5:25  warning  'sdk' should be listed in the project's dependencies. Run 'npm i -S sdk' to add it        import/no-extraneous-dependencies
  8:17  warning  'sdk' should be listed in the project's dependencies. Run 'npm i -S sdk' to add it        import/no-extraneous-dependencies

/Users/pdehaan/dev/github/mozilla/security-advisor-shield-study/lib/TelemetryLog.js
  1:23  warning  'sdk' should be listed in the project's dependencies. Run 'npm i -S sdk' to add it  import/no-extraneous-dependencies

/Users/pdehaan/dev/github/mozilla/security-advisor-shield-study/lib/variations.js
  2:21  warning  'sdk' should be listed in the project's dependencies. Run 'npm i -S sdk' to add it  import/no-extraneous-dependencies

/Users/pdehaan/dev/github/mozilla/security-advisor-shield-study/index.js
  3:23  warning  'lib' should be listed in the project's dependencies. Run 'npm i -S lib' to add it  import/no-extraneous-dependencies

✖ 17 problems (0 errors, 17 warnings)
```

Even reading through the [rule source](https://github.com/benmosher/eslint-plugin-import/blob/master/src/rules/no-extraneous-dependencies.js) I can't see if there is a way to ignore certain dependencies since I know they're not npm-based.
Possibly solutions within `https://github.com/benmosher/eslint-plugin-import/issues/275` (like explicitly disable the ESLint rule for those specific 17 lines instead of disabling globally).